src/components/CookiesDialog.vue

```diff
@@ -0,0 +1,101 @@
+<!-- eslint-disable vue-i18n/no-raw-text -->
+<template>
+  <div
+    class="cookies-dialog"
+    @click.stop
+  >
+    <img src="../assets/cookieImg.svg">
+    <div class="info">
+      Superhero.com respects your privacy. Third-party tracking cookies are blocked by default.
+      In order to play content such as video or audio
+      you have to allow third-party cookies by content provider.
+    </div>
+    <div>
+      <button
+        class="cookies-button"
+        @click="allow"
+      >
+        Allow {{ scope }}
+      </button>
+      <RouterLink
+        v-if="address"
+        class="cookies-button"
+        :to="`/user-profile/${address}`"
+      >
+        Customize policy
+      </RouterLink>
+    </div>
+  </div>
+</template>
+
+<script>
+import backendAuthMixin from '../utils/backendAuthMixin';
+
+export default {
+  mixins: [backendAuthMixin()],
+  props: { address: { type: String, default: '' }, scope: { type: String, required: true } },
+  methods: {
+    async allow() {
+      if (this.address) {
+        await this.backendAuth(`setCookies${this.scope}`, {
+          scope: this.scope,
+          status: 'ALLOWED',
+        });
+      }
+      this.$store.commit('setCookiesListEntry', {
+        scope: this.scope,
+        status: 'ALLOWED',
+      });
```

Comment:
This duplicates several times, let's extract it to a vuex action.
A similar approach is used in https://github.com/aeternity/superhero-ui/pull/817/files#diff-28fe5178003d509da7307f06037a16678fa6faf117b8b4b27ec5bedc74178d10R132-R137